### ISSUE TITLE
BCNY Arc Browser's GPU Acceleration Decimates Performance in Warzone

### ISSUE THREAD

**Issue by JohnnyTex** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#917]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

BCNY Arc Web Browser

### Processor / Processor Number [Required]

i7-13700 (Microcoded) (APO Enabled)

### Graphic Card [Required]

Intel ARC A770 LE

### GPU Driver Version [Required]

32.0.101.6332_101.6253

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [x] Not applicable

### Windows Build [Required]

Windows 11 23H2

### Intel System Support Utility report

[ssubcny.txt](https://github.com/user-attachments/files/18258503/ssubcny.txt)

### Description and steps to reproduce [Required]

When using the GPU Acceleration option for the Arc web browser, fps in Warzone/Black Ops 6 gets crushed. I don't think this was the case for any other game.

**Comment by JohnnyTex** [comment]
Apologies, I left out a crucial detail: this only happens while a video is playing in the background.

**Comment by JohnnyTex** [comment]
This is also the case in COD Cold War.

**Comment by JohnnyTex** [comment]
This is also the case in Halo Infinite.

**Comment by mujeeb-intel** [comment]
Hey @JohnnyTex so Have the hardware acceleration turned ON in google chrome, play a video on youtube and Start playing COD Cold War? Do i have the replication issue steps right? or instead of COD i could try with halo infinite too

**Comment by JohnnyTex** [comment]
Yeah that sounds right. I've attached the setting I have in Arc (based on Chrome). You could do this in Warzone, Cold War, and Halo. 
![Screenshot 2025-01-07 152643](https://github.com/user-attachments/assets/a3f24dcc-69e0-4ae0-b116-6561fb76a273)

**Comment by mujeeb-intel** [comment]
awesome, yeah same settings in my chrome and Edge. Arc browser is however new to me but nice to learn new things. Hang tight ill do some testing and let you know.

**Comment by mujeeb-intel** [comment]
Hello @JohnnyTex I tried Halo infinite and cold war and was not able to replicate your issue. Both games worked as expected. Attached are my screenshots of the performances with arc and youtube videos running and playing the games and without.

Specs: 1440p w/ A770

![Image](https://github.com/user-attachments/assets/3443c184-7188-4635-95c9-2df386d488de)

without arc
![Image](https://github.com/user-attachments/assets/75660675-a56d-4f6a-ab00-277ec9c2e43c)

with arc

![Image](https://github.com/user-attachments/assets/68b35272-fb6a-418b-8998-d5b93a91d0a9)

**Comment by JohnnyTex** [comment]
I'll see about testing things out this weekend. Will probably be doing so on the newest driver too.

**Comment by mujeeb-intel** [comment]
I have tried both the latest one released today and the one before. Both worked fine (you can check out my perf graphs) the issues seem to be isolated to your platform. Try running the DDU and then check if it resolves it. Please let me know.

**Comment by JohnnyTex** [comment]
Just tried it again in Cold War after DDU'ing the system and clean installing the newest driver, it's still happening. Will upload the video evidence a bit later.

**Comment by mujeeb-intel** [comment]
Hello @JohnnyTex Just tried on another system and it works the same as expected like the perf charts I shared.

**Comment by JohnnyTex** [comment]
No dice. As you can see in this video, when I press play on my keyboard and the background video begins playing, the game's fps tanks. 

https://drive.google.com/file/d/1PxVQx5pFdkbTISWbYniCyuWli596OXkP/view?usp=sharing

**Comment by JohnnyTex** [comment]
I see the effects in Fortnite too, though that game still runs fine for the most part. It's just that the 1% lows take a nasty hit and thus the perf can be very inconsistent.

**Comment by JohnnyTex** [comment]
Actually I think this is in every game I play, cause it's even somewhat noticeable in Street Fighter 6. Given that that game is the only game seemingly to work w/ IGS' perf tab for me atm, I took a look at the telemetry and found that curiously, the 1% lows were _higher_ than what the fps cap in the game is supposed to allow it to be. 

![Image](https://github.com/user-attachments/assets/c08a76ef-c859-47e1-8fb9-c1743e055352)

**Comment by JohnnyTex** [comment]
![Image](https://github.com/user-attachments/assets/b2f1320f-b463-494c-825b-4fdb6e0ce0b8)

**Comment by JohnnyTex** [comment]
This is another curious finding; for some reason in Armory Crate it's reporting my GPU is over 100% utilized.

**Comment by JohnnyTex** [comment]
I'm actually under the impression that Low Latency Mode + Boost was actually the cause. Which is very weird.

**Comment by Felipe-Intel** [comment]
@JohnnyTex there's currently an issue where videos played in Chrome have massive frame drops when LL is enabled. You may want to double check if it is indeed the LL setting affecting your browser and this in turn is affecting your game performance.

**Comment by mujeeb-intel** [comment]
@JohnnyTex any findings?

**Comment by JohnnyTex** [comment]
Sorry didn't do that this past week, will hopefully get to it tonight.

**Comment by JohnnyTex** [comment]
That didn't happen, but I'm gonna try and set some time aide for it this week fs.

**Comment by mujeeb-intel** [comment]
Hello  @JohnnyTex  ,

After re-watching the video you shared and mentioning it's happening on majority of the games you play, it indicates to me that something on your platform may be conflicting with the game's performance. I'm not sure what the issue could be but my suggestion would be to try the following steps:

1. Turn off all of the startup applications from the task manager
2 Test it using only Chrome, Edge and Firefox since these are officially supported browsers on this graphics card.
3. Install fresh copy of windows. 
 
since other systems don't exhibit the issue you're describing, it's going to be super difficult for me not only to diagnose the issue, but to submit this to the drivers team without any actionable data.

**Comment by JohnnyTex** [comment]
So something to note is I can positively confirm that regular Low Latency doesn't cause these issues. I've had it on for some time now and there the fps has been perfectly fine w/ or w/o videos on.

**Comment by JohnnyTex** [comment]
I think something's up w/ LL + Boost specifically here, bc I turned it back on, restarted my system, and the behavior came right back up. Once I turned it off though everything was fine again.

**Comment by mujeeb-intel** [comment]
Hey @JohnnyTex I know its been a while, but I wanted to come back and let you know, I have had multiple people test the issue specifically how you're describing it, by running the browser and no one has been able to reproduce this issue. We tried with LL+Boost and restart On/OFF and did not see any perf hit. However, there is another reported issue but thats for the performance overall in general with the game so I am looking into that one.

**Comment by JohnnyTex** [comment]
> Hey [@JohnnyTex](https://github.com/JohnnyTex) I know its been a while, but I wanted to come back and let you know, I have had multiple people test the issue specifically how you're describing it, by running the browser and no one has been able to reproduce this issue. We tried with LL+Boost and restart On/OFF and did not see any perf hit. However, there is another reported issue but thats for the performance overall in general with the game so I am looking into that one.

I'll have to do more testing but for whatever reason resetting my PC seemed to fix this issue. I have no idea why that would be, but it is what it is. There now seems to be a far more localized (and minor) issue that's related to this, but I'll do some more testing to see if it's truly a problem or not.
